### PR TITLE
Feature/BRey/FIX-Documentation

### PR DESCRIPTION
**Pull request by BraianRey**
Fix or add the documentation of the classes, add the plugin to control the generation of the documentation in the pom.xml of the projects. Use "mvn javadoc:javadoc" in terminal to generate it

### CODE REVIEW

**Review by BraianRey — COMMENTED** [review]
Other poms are changed for this version

File: payment-service/src/main/java/co/unicauca/payment/domain/service/PaymentFacade.java [hunk 1]
```diff
@@ -7,6 +7,9 @@
 import co.unicauca.payment.domain.Account;
 import co.unicauca.payment.domain.Transaction;
 
+/**
+ * Clae PaymentFacade
```
Comment (Einx001) [inline payment-service/src/main/java/co/unicauca/payment/domain/service/PaymentFacade.java:11]: *clase

Comment (BraianRey) [reply payment-service/src/main/java/co/unicauca/payment/domain/service/PaymentFacade.java]: Fixed

**Review by Jojan-Esteban-Serna — APPROVED** [review]

**Review by SantiAV25 — APPROVED** [review]
sweeet

**Pull request merged**